Tests/WorkspaceTests/InitTests.swift

```diff
@@ -266,10 +266,15 @@ class InitTests: XCTestCase {
             try localFileSystem.removeFileTree(packageRoot)
             try localFileSystem.createDirectory(packageRoot)
 
+            let packageTemplate = InitPackage.PackageTemplate(sourcesDirectory: RelativePath("./Sources"),
+                                                              testsDirectory: RelativePath("./Tests"),
```

Comment:
Please do not assume that the path separator is `/`.  You should use the `join` method to generate the relative paths.